monitor-data/custom-webhook-notifier.mdx

```diff
@@ -48,4 +48,23 @@ The example below is a custom webhook notification generated from a threshold mo
     "matchedEvent": null
   }
 }
+```
+
+The example below formats the webhook message to match the [expectations of incident.io](https://api-docs.incident.io/tag/Alert-Events-V2/) using the monitor ID as the `deduplication_key`.
```

Comment:
```suggestion
The example template below formats the webhook message to match the [expectations of incident.io](https://api-docs.incident.io/tag/Alert-Events-V2/) using the monitor ID as the `deduplication_key`.
```

Small change to make it clear that we're sharing examples of two different things here — the first is example data that would be _applied to_ a template, the second is an actual example template.